ccm_web/tsconfig.json

```diff
@@ -10,6 +14,7 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "skipLibCheck": true,
```

Comment:
The `@nestjs` starter project (https://github.com/nestjs/typescript-starter) uses a library check (using the `skipLibCheck`default of `false`). When I tried building for prod with the default, there were a number of errors thrown, mostly from `webpack`-related types. The idea is that TypeScript checks the consistency of all the declaration files used by your project. TypeScript recommends `true` (I believe for performance reasons), but we should explore this a little more. I will open a new issue.

Reply:
See #76.